CONTRIBUTING.md

```diff
@@ -96,10 +96,21 @@ thought of as being the motivation for your change.
 
 Describe the modifications you've made.
 
+If this PR changes a behavior, it is helpful to describe the difference between
+the old behavior and the new behavior. Providing samples of before and after
```

Comment:
I might replace "Providing samples..." with "Provide before and after screenshots, CLI output, or YAML where applicable" or something to that effect, to flesh out what "samples" are